src/main/java/core/basesyntax/strategy/operation/ReturnOperation.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.strategy.operation;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void performOperation(FruitTransaction transaction) {
+        int currentQuantity = Storage.storage.get(transaction.getFruitName());

```

Comment:
Make sure to check if the fruit is already present in the storage before getting its quantity. If it's not present, you should initialize it with a quantity of 0 to avoid a NullPointerException.